ibm/mas_devops/roles/suite_app_config/files/manage/manage-additional-server-config.xml.j2

```diff
@@ -18,6 +18,6 @@
     <jmsQueue jndiName="jms/maximo/int/queues/notferr"><properties.wasJms queueName="notferrbd"/></jmsQueue>
     <jmsQueue jndiName="jms/maximo/int/queues/testqueue"><properties.wasJms queueName="testqueuebd"/></jmsQueue>
 
-    <jmsActivationSpec id="maximoui/mboejb/JMSContQueueProcessor-1" maxEndpoints="5"><properties.wasJms destinationLookup="jms/maximo/int/queues/cqin" maxConcurrency="5" maxBatchSize="20" connectionFactoryLookup="jms/maximo/int/cf/intcf"/></jmsActivationSpec>
-    <jmsActivationSpec id="maximoui/mboejb/JMSContQueueProcessor-2" maxEndpoints="1"><properties.wasJms destinationLookup="jms/maximo/int/queues/cqinerr" maxConcurrency="1" maxBatchSize="20" connectionFactoryLookup="jms/maximo/int/cf/intcf"/></jmsActivationSpec>
+    <jmsActivationSpec id="{earFileName}/mboejb/JMSContQueueProcessor-1" maxEndpoints="5"><properties.wasJms destinationLookup="jms/maximo/int/queues/cqin" maxConcurrency="5" maxBatchSize="20" connectionFactoryLookup="jms/maximo/int/cf/intcf"/></jmsActivationSpec>
```

Comment:
Should this be {{ earFileName }} rather than {earFileName} ?

Reply:
That uses the module `replace` module which uses a given string to match a regular expression:
https://github.com/ibm-mas/ansible-devops/blob/fix-manage-storage-classes/ibm/mas_devops/roles/suite_app_config/tasks/manage.yml#L45

I'll just use a better matching string to not cause confusions.